REPOSITORY.md

```diff
@@ -0,0 +1,39 @@
+# Open Cluster Management Repositories Guidelines
+
+This document attempts to outline a structure for creating GitHub repositories with the Open-Cluster-Management project. It also describes how and when repositories are removed.
+
+# Rules for New Repositories
+
+* All repos will live in github.com/open-cluster-management-io/\<project-name\>.
+* Must contain the topic for the sponsoring sub projects - e.g. application. (Added through the Manage topics link on the repo page.)
+* Must adopt the Open-Cluster-Management Code of Conduct
+* All code projects use the Apache License version 2.0.
+* All OWNERS of the project must also be active Open-Cluster-Management members.
+* Must be approved by the following process.
+  1. It has been discussed and recognized in the community meeting with a
+   publicly linkable written decision.
+  2. It must have two sponsors, each of whom must be the maintainer of a sub project.
+  3. An repository onboarding issue is created in the community repo.
+
+## Removing Repositories
```

Comment:
removing sounds like an irrevocable process that completely eliminates the repo, while in kubernetes the sun-setting repos will end up being transferred to https://github.com/kubernetes-retired. i think just explicitly freezing the repo for those project we're not going to move forward will be fine?